src/components/monitor/FeatureSection.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import Image from 'next/image';
+import { getImage } from '@/lib/getImage';
+import Button from '../shared/Button';
+
+import { useRouter } from 'next/router';
+import { HaveIbeenPwnedText } from '@/assets/types/have-i-been-pawned';
+
+export interface FeatureSectionProps {
+  textContent: HaveIbeenPwnedText['FeatureSection'];
+}
+const CardText = ({ textContent }) => {
```

Comment:
Type the props using an interface or 
````
{ textContent } : { textContent: HaveIBeenPwnedText['FeatureSection'][...]
````

Also, I think there are a shared Card component. If it exists, you can use it.

Reply:
Changed to the shared card component